src/common/components/Tag/Tag.theme.ts

```diff
@@ -0,0 +1,66 @@
+import { type VariantProps, tv } from "tailwind-variants";
+
+export type TagVariants = VariantProps<typeof tagTheme>;
+
+export const tagTheme = tv(
+  {
+    slots: {
+      tag: [
+        "inline-flex",
+        "min-w-14",
+        "py-[0.125rem]",
+        "px-3",
+        "gap-2",
+        "justify-center",
+        "items-center",
+        "rounded-[6.1875rem]",
+        "border",
+        "border-solid",
+        "border-border-default",

```

Comment:
I think we can also add some transition for the background changes `transition-color duration-300 ease-in-out`